### ISSUE TITLE
Sliders in sidebar-selection-info are too wide so scrollbar always shows

### ISSUE THREAD

**Issue by punchcutter** [googlefonts/fontra#300]
When viewing rcjk files and looking at deep components the y scrollbar is always on in sidebar-selection-info. 

<img width="285" alt="slider-scrollbar-on" src="https://user-images.githubusercontent.com/11257204/220862802-78f449b8-76fb-49c6-b8d4-be3c36b7e22e.png">

If I turn it off manually it still seems really tight on the right.

<img width="306" alt="slider-scrollbar-off" src="https://user-images.githubusercontent.com/11257204/220862805-2b9bf4d2-d058-4ede-87da-92b639a32826.png">

With max value the sliders are cut off even when I turn off the scrollbar.

<img width="273" alt="slider-scrollbar-maxvalue" src="https://user-images.githubusercontent.com/11257204/220863852-b9b81ab0-2411-4206-9336-889840400298.png">

I also constantly try to expand the sidebar like I would expect possible in many apps so that would be nice. If I want to use the axis sliders to adjust anything I only have a tiny space to work with. Of course that's not a huge deal so could be a feature request, but the cut off and constant scrollbar has been bugging me.

**Comment by justvanrossum** [comment]
May relate to #293.

**Comment by justvanrossum** [comment]
Out of curiosity, on which platform and with which browser are you seeing this?

**Comment by punchcutter** [comment]
Chrome on Mac.

**Comment by punchcutter** [comment]
Checked Safari and Firefox and same thing.

**Comment by justvanrossum** [comment]
What is your global scroll bar setting? (System Prefs -> General)
<img width="487" alt="image" src="https://user-images.githubusercontent.com/4246121/221128672-a8258081-cc64-4b4c-8747-1f05a34b2742.png">

In the past I think I've seen this setting also being influenced by the presence of a Wacom tablet.

**Comment by justvanrossum** [comment]
I think I can reproduce your observations if I switch to "Always".

**Comment by justvanrossum** [comment]
Solutions:

- Make the layout work even when scrollbars are always visible

Or

- Override the OS setting and always hide the scrollbars (if that's even possible with CSS)

**Comment by punchcutter** [comment]
Oh duh, I forgot about that setting. I had it on "Always" and if I turn it off it looks fine. I think I turned it on like 20 years ago and forgot about it. I think the main reason I have that on is because in general hidden scrollbars are unreliable and waste my time trying to get the mouse (or Wacom more often for me) to hover over and make it appear. So I set it to "Always" long ago and never noticed any issue until this.
